spec/truffle/securerandom_spec.rb

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved. This
+# code is released under a tri EPL/GPL/LGPL license. You can use it,
+# redistribute it and/or modify it under the terms of the:
+#
+# Eclipse Public License version 1.0
+# GNU General Public License version 2
+# GNU Lesser General Public License version 2.1
+
+require_relative '../ruby/spec_helper'
+
+describe "Loading SecureRandom" do
+
+  it "should not eagerly load OpenSSL" do
+    ruby_exe("print defined?(OpenSSL)", options: "-rsecurerandom").should == ""
```

Comment:
I would use `p` so we can test against `"nil\n"` which is a bit clearer and shows the sub-process didn't just crash or made no output.

Reply:
Fair enough on the crash part. But I don't think having to match against "\n" helps legibility that much.

Reply:
Yeah, the `\n` is not-useful side-effect of using `p`. `p` is used a lot in ruby/spec like that though.